config/locales/en.yml

```diff
@@ -1,33 +1,43 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t "hello"
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t("hello") %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   "true": "foo"
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  articles: 'Articles'
+  time:
+    formats:
+      short: 'around %H'
+      long: 'Current time is: %H:%M:%S'
+  routes:
+    articles: 'Back to articles'
+  article_pluralization:
+    zero: 'There is no article.'
+    one: 'There is one article.'
+    other: 'There are %{count} articles.'
+  article:
+    create: 'Create new article'
+    new_article: 'New Article'
+    edit_article: 'Edit Article'
+    edit: 'Edit'
+    delete: 'Destroy'
+  article_form:
```

Comment:
A lot of "chapters" related to articles (articles, article_pluralization, article, article_form), Better to organise them under one "chapter" `article`. Same for Comments.
Plus, where translations for validation errors?